uPortal-security/uPortal-security-authn/src/main/java/org/apereo/portal/security/provider/SimpleSecurityContext.java

```diff
@@ -34,9 +34,7 @@
 
     private final int SIMPLESECURITYAUTHTYPE = 0xFF02;
 
-    SimpleSecurityContext() {
-        super();
-    }
+    /* package-private */ SimpleSecurityContext() {}
```

Comment:
It is.

It's also arguably the least used, least commonly appropriate access level in Java.  Many practitioners (including myself) believe it was a poorly-chosen default.  (It's too late to change it now.)  Many Java programmers (again incl. me) imagine that the package-private access level is commonly used by mistake, since it is the default.  For this reason, many Java developers add a comment when they use it to indicate that they actually intended to use it.